adoc/admin-disaster-recovery-backup.adoc

```diff
@@ -0,0 +1,161 @@
+= Backup
+This chapter describes the backup of {productname}.
+
+== Data To Backup
+* The `skuba`: command-line binary for the running cluster, use to replace nodes from cluster.
+* The skuba cluster folder: directory where used for running `skuba` commands. Contains cluster configurations and authentication files.
+* The kubernetes folder of every node in cluster: directory contains kubernetes configurations.
++
+```
+/etc/kubernetes
+```
+* The kubelet folder of every node in cluster: directory contains kubelet configurations.
++
+```
+/var/lib/kubelet
+```
+* The etcd cluster database: holds all cluster data. Can be used to recover master nodes.
+
+== Create Etcd Cluster Database Backup
+=== Procedure
+1. Mount external storage device to all master nodes. This is only required if following step is using local hostpath as volume storage.
+2. Create backup.
++
+HINT: Backup size depends on the cluster. Ensure each of the backup has sufficient space. The available size should be more than the database snapshot file. You should also have a rotation method to clean up the unneeded snapshot overtime. When there is insufficient space available during backup, pod will fail to be in `Running` state and `no space left on device` error will show in pod logs.
```

Comment:
Seem like asciidoc does not support HINT https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/

Reply:
Yes I don't know if we have a consistent way to write a HINT. Currently I'm leaving this for the doc team to fix. I will add a TODO note for doc team.